pkg/exchange/okex/convert.go

```diff
@@ -223,20 +219,20 @@ func toGlobalOrders(orderDetails []okexapi.OrderDetails) ([]types.Order, error)
 	return orders, nil
 }
 
-func toGlobalOrderStatus(state okexapi.OrderState) (types.OrderStatus, error) {
+func toGlobalOrderStatus(state okexapi.OrderState) types.OrderStatus {
```

Comment:
is there a reason to remove the error?

Reply:
No, No need to remove the error. Sorry, I used other exchange as reference, right now I use toGlobalOrders() as reference.